src/acceleration/config/AccelerationConfiguration.cpp

```diff
@@ -87,25 +87,29 @@ void AccelerationConfiguration::connectTags(xml::XMLTag &parent)
   // static int recursionCounter = 0;
   // recursionCounter++;
 
-  XMLTag::Occurrence occ = XMLTag::OCCUR_NOT_OR_ONCE;
-  std::list<XMLTag>  tags;
+  XMLTag::Occurrence  occ = XMLTag::OCCUR_NOT_OR_ONCE;
+  std::vector<XMLTag> tags;
   {
     XMLTag tag(*this, VALUE_CONSTANT, occ, TAG);
+    tag.setDocumentation("Accelerates coupling data with constant underrelaxation.");
     addTypeSpecificSubtags(tag);
     tags.push_back(tag);
   }
   {
     XMLTag tag(*this, VALUE_AITKEN, occ, TAG);
+    tag.setDocumentation("Accelerates coupling data with dynamic underrelaxation.");
```

Comment:
Should we mention it is Aitken relaxation? 
"Accelerates coupling data with dynamic Aitken under-relaxation."